pcdora.info.yml

```
@@ -3,3 +3,7 @@ type: module
 description: 'Providence College Customizations'
 core_version_requirement: ^8 || ^9 || ^10
 package: 'discoverygarden'
+dependencies:
+ - drupal:migrate
+ - drupal:dgi_standard_oai
+ - drupal:rest_oai_pmh
```

Neither of these are core Drupal modules (as the `drupal:` signifies). See [the docs](https://www.drupal.org/docs/develop/creating-modules/let-drupal-know-about-your-module-with-an-infoyml-file#s-complete-example) for more details.

```suggestion
 - dgi_standard_oai:dgi_standard_oai
 - rest_oai_pmh:rest_oai_pmh
```

Should probably also be in the `composer.json` in some capacity?